testing/lightspeed/command_args.yaml

```
@@ -0,0 +1,24 @@
+{% set secrets_location = false | or_env(secrets.dir.env_key) %}
+{% if not secrets_location %}
+  {{ ("ERROR: secrets_location must be defined (secrets.dir.name="+ secrets.dir.name|string +" or env(secrets.dir.env_key=" + secrets.dir.env_key|string + ")) ") | raise_exception }}
+{% endif %}
+{% set s3_ldap_password_location = secrets_location + "/" + secrets.s3_ldap_password_file %}
+
+# ---
+
+rhods deploy_ods:
+  catalog_image: {{ rhods.catalog.image }}
+  tag: {{ rhods.catalog.tag }}
+  channel: {{ rhods.catalog.channel }}
+  version: {{ rhods.catalog.version }}
+
+wisdom_deploy_model:
```

```suggestion
wisdom deploy_model:
```
the main key name in this file should be the `<group> <command name>`, not the `<role name>`